code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -157,7 +157,7 @@
 	//Taken from update_icons
 	var/list/overlays_standing[TOTAL_LAYERS]
 	var/hardcore = FALSE //If TRUE, removes the body upon unrevivable death (for WO)
-	appearance_flags = KEEP_TOGETHER | TILE_BOUND
+	appearance_flags = KEEP_TOGETHER|TILE_BOUND|PIXEL_SCALE|LONG_GLIDE
```

Comment:
how does this change factor into this PR? does making the officer headshot not work without it?

Reply:
original author chipped in; PIXEL_SCALE likely necessary for the portraits to look right, LONG_GLIDE is not and should be removed.